### DISCUSSION TITLE
many duplicate entries

### DISCUSSION THREAD

**Discussion by Suedschwede66** [Arnold-n/P1P2MQTT#147] [Q&A]
Hello Arnold-n,
many duplicate entries in HC_FieldSettings, please look example

![Bildschirmfoto vom 2024-12-03 17-32-21](https://github.com/user-attachments/assets/e63b3634-e9f8-4b5b-af0b-480364c121d4)

**Comment by Arnold-n** [comment]
Thanks for noting. I guess the duplicates are the ones that were changed manually? I need to ensure those are not separately reported to HA, will do so in the next version.

**Reply by Suedschwede66** [reply]
The entities are created twice for whatever reason.
i use last Firmware P1P2MQTT-bridge-v0.9.58rc2-Daikin-E.bin

![Bildschirmfoto vom 2024-12-03 18-33-57](https://github.com/user-attachments/assets/409bce2f-df5b-43eb-ade3-2731ef307a10)
![Bildschirmfoto vom 2024-12-03 18-33-14](https://github.com/user-attachments/assets/fe0a8610-b6ad-4fed-bffb-b2d504675831)

**Comment by Arnold-n** [comment]
I hope v0.9.58rc4 resolves all duplicate entries and does not introduce any new issues. After installing, `MQTT_Delete_Own_Rebuild` (/D12) is needed to remove/replace some of the old entries.

**Reply by Suedschwede66** [reply]
All ok now thanks.
